engine_modules/detroit_inc/models.py

```diff
@@ -99,6 +100,7 @@ class DIncVoteSession(models.Model):
 	Used to keep track of the current Detroit, Inc line
 	"""
 
+	# @Neamar: DIncVoteORder.DINC_COALITION_CHOICES contains tuples, not strings, should it not be c[1] for c in DINC_COALTION_CHOICES instead ?
```

Comment:
nope, choices can take as argument tuples (it will be nicely formatted in the admin)